### PR TITLE
Finally updates the sec sprites from #12250

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
#12250 was meant to update our newbluesec sprites a tad, giving the HoS an especially nice gold compared to the current one.
image

This never came to be due to people choosing rushed jobs over just sorting it correctly;
I'd JUST finished my PR to split security uniforms into a unique .dmi instead of one big under.dmi. They were not put into this new, clean and sorted .dmi.

This PR fixes that. I dont expect to do another one like this.
### Clothing sprites should go into their sorted files if applicable. More will soon be made.

While I was touching the security sprites, I also touched up the Sol Police sprites to be modern and clean:
image

And removed unused Police sprites and the redundant Traffic uniform which was just a slightly-different-badge-color from the normal Police uniform.
And changed their descriptions a tad, especially the oddly structured parts.

AND fixed the HoS formal suit jacket having the wrong icons in the wrong places. Now it'll actually give a worn sprite instead of the mob sprite.

## How This Contributes To The Skyrat Roleplay Experience
Sprite cleanup is imporpampt.

## Changelog

:cl:
fix: fixed the HoS Formal Jacket having swapped mob/obj icons. Now you can wear it correctly instead of only one direction!
fix: fixed some improperly updated security sprites. They are now properly updated, namely the HoS and Warden uniforms.
imageadd: added new Sol Police uniform sprites to fit better in the modern sprite era.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Need to touch the digi ones before this is ready

**Comment by ghost** [comment]
Oh god bless my laziness there's still no security_digi.dmi
That'll be a separate PR 🦀 This ones ready to go

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 1]
```diff
@@ -52,30 +44,34 @@
...
 	icon_state = "policecadetalt"
 
-/obj/item/clothing/under/rank/security/peacekeeper/sol/traffic
-	name = "sol traffic police uniform"
-	desc = "A light blue shirt with navy pants, perfect for standing and shouting at cars."
-	icon_state = "policetrafficalt"
-///OFFICERS
 /obj/item/clothing/under/rank/security/peacekeeper/sol
 	name = "sol police uniform"
-	desc = "A light blue shirt with navy pants, perfect for opressing the Underclasses, like Catgirls."
```
Comment (theselfish) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm:68]: Catgirl joke removed. Distressing.

Comment (ghost) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:68]: Yes, but the new one is much more accurate to the general-uniformed officer IMO. Plus "Catgirls" is too offputting and the word "Underclasses" feels gross

Comment (theselfish) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:68]: Let's get a third opinion from the Cattainer?
@tf-4 Thoughts?

Comment (tf-4) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:68]: Stealth felinid buff. Approved.

**Pull request merged**